packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.html

```diff
@@ -1,40 +1,87 @@
+                             <c-quantic-category-facet-parent onselectvalue={onSelect} key={v.value} item={v}></c-quantic-category-facet-parent>
+                         </template>
+                         <template if:true={activeParent}>
+                             <li class="slds-var-m-horizontal_medium slds-grid">
+                                 <span class="nowrap parent-text__bold">{activeParent.value} ({activeParent.numberOfResults})</span>
+                             </li>  
+                         </template>
+                         <li class="slds-var-m-horizontal_medium slds-grid">
+                            <ul>
+                             <template for:each={values} for:item="v">
+                                <c-quantic-category-facet-value 
+                                    onselectvalue={onSelect} 
+                                    key={v.value} 
+                                    item={v} 
+                                    data-cy={v.value}>
+                                </c-quantic-category-facet-value>
+                             </template>
+                            </ul>
+                         </li>
+                    </template>
+                    <template if:true={isFacetSearchActive}>
+                        <template if:true={hasSearchResults}>
+                            <template for:each={facetSearchResults} for:item="result">
+                                <c-quantic-category-facet-value 
+                                    onselectvalue={onSelect} 
+                                    key={result.value}  
+                                    item={result} 
+                                    withsearch="true">
+                                </c-quantic-category-facet-value>
+                            </template>
+                        </template>
+                        <template if:false={hasSearchResults}>
+                            <li class="slds-var-m-around_small slds-grid">
+                                <span class="slds-align_absolute-center slds-var-p-vertical_small slds-var-m-bottom_medium slds-var-p-horizontal_xx-large facet__search-results_no-match">{noMatchesForLabel}</span>
+                            </li>
+                        </template>
+                        <template if:true={canShowMoreSearchResults}>
+                            <li class="slds-var-p-left_small slds-var-p-bottom_medium">
+                                <span class="facet__search-results_more-matches">{moreMatchesForLabel}</span> 
+                            </li>
+                        </template>
+                    </template>
+                </ul>
+                <template if:true={canShowLess}>
+                    <button variant="base" label={labels.showLess}
+                      class="slds-var-m-left_x-small facet__show-less" onclick={showLess}>
+                      <span class="facet__show-less-text">{labels.minus} {labels.showLess}</span>
```

Comment:
As mentioned in other comments, I would suggest to include the minus sign directly in the `showLess` label. With the actual markup, a customer that sets the `minus` label to an empty string will still get a leading whitespace before the `showLess` label. Having to alter the facet code to remove that space would be unpleasant.

Reply:
fixed